apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -9,10 +9,20 @@ import { keyBy, map, has } from 'lodash';
 // TODO: remove once we have proper previews from API
 if ( window.starterPageTemplatesConfig ) {
 	const PREVIEWS_BY_SLUG = {
-		home: 'https://starterpagetemplatesprototype.files.wordpress.com/2019/05/starter-home-2.png',
-		menu: 'https://starterpagetemplatesprototype.files.wordpress.com/2019/05/starter-menu-2.png',
-		'contact-us':
-			'https://starterpagetemplatesprototype.files.wordpress.com/2019/05/starter-contactus-2.png',
+		home: {
+			src: 'https://starterpagetemplatesprototype.files.wordpress.com/2019/05/starter-home-2.png',
+			alt:
+				'Full width hero banner, followed by alternating text and image sections, followed by a Get in Touch area',
```

Comment:
If we want to provide a meaningful alt text for every preview image, it would probably require more planning to properly scale that? Should we keep it empty for now and treat the previews as decorative?

Reply:
> Should we keep it empty for now and treat the previews as decorative?

Yes we _can_ do. I fear however, that if we don't put something in place then it will be too easy to just forget about making this feature a11y to non-sighted users. 

Is _enabling_ the ability to define `alt` text not the lowest barrier to entry here? Obviously we will not be hard coding it and the data will ultimately come from the API. Given that we already have the ability to add `alt` text to images within the WP UI, it seems like a good step in the right direction.

Reply:
I love this idea! What about filing an issue to talk more about this for future versions? There is a potential to auto-generate it based on some super simple content analysis for example.

Reply:
Let's grab the alt text associated with the post thumbnail then. Just be prepared that that will most likely be empty

Reply:
> Just be prepared that that will most likely be empty

👍 https://github.com/Automattic/wp-calypso/pull/33408/files/597866aba2057035fee25c25e2b2754ac1439e75#diff-e78e4b83473f3ff5bbcab357be669290R52